istio.deps

```diff
@@ -4,13 +4,13 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "a166ec637278ca2495c677ac32d3c53bbc5f3a23"
+    "lastStableSHA": "12cfbda324320f99e0e39d7c393109fcd824591f"
```

Comment:
This doesn't look right to me based on the title.

Reply:
I used the `update_deps.sh` script to update it, which updates both PROXY and CNI sha. I can limit it to just the CNI SHA.

Reply:
I think it's ok to include the proxy SHA in here. Thanks.